src/new/wallet.js

```diff
@@ -138,6 +139,10 @@ class HathorWallet extends EventEmitter {
     this.pinCode = pinCode;
     this.password = password;
 
+    if (preCalculatedAddresses) {
```

Comment:
I don't think we need this `if` since the default value is `null`

Reply:
The idea was to avoid clutter on the instance object, but I agree that's a good idea. Done on [5c74a59](https://github.com/HathorNetwork/hathor-wallet-lib/pull/354/commits/5c74a595c99e414d85ee32b924b2159c0e12f5b7).